examples/azure/packages/.yarnrc.yml

```diff
@@ -1 +1 @@
-nodeLinker: node-modules
+nodeLinker: "node-modules";
```

Comment:
Still should be without semicolon:

```
nodeLinker: node-modules
```

Reply:
done

Reply:
should this be `nodeLinker: 'node-modules'` or just `nodeLinker: node-modules`?

Reply:
@thiemogries I guess both would be valid YAML, with quotes it's for sure less error-prone.

Reply:
that was my thought as well. what do you think @thiemogries